server/command_create.go

```diff
@@ -35,18 +50,27 @@ func parseCreateArgs(args []string, install *Installation) error {
 		return err
 	}
 
-	install.Size, err = createFlagSet.GetString("size")
+	install.Size, err = createFlagSet.GetString(size)
+	if err != nil {
+		return err
+	}
+	install.Version, err = createFlagSet.GetString(version)
 	if err != nil {
 		return err
 	}
-	install.Version, err = createFlagSet.GetString("version")
+
+	install.EnterpriseTag, err = createFlagSet.GetString(enterpriseTag)
 	if err != nil {
 		return err
 	}
 
+	if install.EnterpriseTag == "" {
+		install.EnterpriseTag = install.Version
+	}
```

Comment:
Apologies, but I am a bit lost about what the purpose of the Enterprise Tag is here. I see that we are fully passing through the value provided by the user if they give one, but then we simply store it with the installation metadata inside the plugin. I don't see how this value actually modifies the installation that is created for the user. Is this PR just prepping for usage of the tag in a later change?

Reply:
yeah that it totally a mistake on my part, i am a bit confused around which image we are building. From mattermost-cloud i can see that we are building `mattermost/mattermost-enterprise-edition` i thought that this was only the image from the enterprise repo but i guess is a bundled one right?

Reply:
That's right, we can only work with a single docker image that is built through external CI/CD. Right now, you can't mix-n-match the webapp, server, and enterprise repo in a completely flexible way.

The way this would be supported is to work to create a CI/CD pipeline that allows you to pick your versions of each repo and then specify the resulting docker image in the cloud plugin.

Reply:
Superb, that is my understanding. Anyone against on loosing up the validation on the version argument? and allow it to support a docker tag?

Reply:
That's exactly how it works right now. :)

You specify a docker tag for `version` and then the plugin keeps track of the exact image sha used at that time.

Reply:
hmm looking more into it, we are using, heroku's docker registry client to check for the validity of a docker tag. So the question is why we are getting invalid that when we are passing a docker tag in the version argument:

`Error: mattermost/mattermost-team-edition:69cbba7 is not a valid docker tag for repository mattermost/mattermost-enterprise-edition`

Reply:
Hmm, I just tried that and I think it worked as you would expect:

```
/cloud create gabe10 --version=69cbba7
{
    "Name": "gabe10",
    "ID": "coyzdiadqtffzpzo4gx7dtbpka",
    "OwnerID": "e9g65ngxs7dntn1z14pchnjjiy",
    "GroupID": "cuappmdoq7rwibpb5nzbtcwgqy",
    "Version": "sha256:ad795aa7cbdd913272bc79a207209711a032b8c5ded7de6c16a8398142de4eab",
    "Image": "mattermost/mattermost-enterprise-edition",
    "DNS": "gabe10.test.mattermost.cloud",
    "Database": "aws-multitenant-rds-postgres",
    "Filestore": "aws-multitenant-s3",
    "License": "hidden",
    "MattermostEnv": null,
    "Size": "miniSingleton",
    "Affinity": "multitenant",
    "State": "creation-requested",
    "CreateAt": 1614012574027,
    "DeleteAt": 0,
    "APISecurityLock": false,
    "LockAcquiredBy": null,
    "LockAcquiredAt": 0,
    "TestData": false,
    "Tag": "69cbba7"
}
```

Reply:
So i wasted enough of your time. The tag needs to be the short-sha of the latest commit. Closing this PR since is pointless

Reply:
No worries at all! To be specific, the version just needs to match any tag you see here: https://hub.docker.com/r/mattermost/mattermost-enterprise-edition/tags